### PR TITLE
Add all Import origin for Aggregator Admin bar group

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#68693](http://central.tri.be/issues/68693)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Admin_Bar.php:36](https://github.com/moderntribe/the-events-calendar/blob/a736deb3e1213f044c874e67a44251bcbd837124/src/Tribe/Aggregator/Admin_Bar.php#L36)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Admin_Bar.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ * @since  TBD
+ */
+class Tribe__Events__Aggregator__Admin_Bar {
+
+	/**
+	 * @since  TBD
+	 *
+	 * @var Tribe__Events__Admin__Bar__Admin_Bar
+	 */
+	protected static $instance;
```
Comment (borkweb) [inline src/Tribe/Aggregator/Admin_Bar.php]: Add a line break between property declarations and docblock comments after them, pretty please!

File: src/Tribe/Aggregator.php [hunk 2]
```diff
@@ -516,6 +516,22 @@ public function add_csv_mimes( $info, $file, $filename, $mimes ) {
...
 
 	/**
+	 * Adds the Items for Aggregator on the Admin bar
+	 *
+	 * @since   TBD
+	 *
+	 * @return  void
+	 */
+	public function add_admin_bar_items() {
+		$admin_bar = Tribe__Events__Aggregator__Admin_Bar::instance();
```
Comment (borkweb) [inline src/Tribe/Aggregator.php]: Should this use di52?

File: src/Tribe/Aggregator/Admin_Bar.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	protected $config;
+
+	/**
+	 * Singleton constructor for the class.
+	 *
+	 * @since  TBD
+	 *
+	 * @return Tribe__Events__Aggregator__Admin_Bar
+	 */
+	public static function instance() {
```
Comment (borkweb) [inline src/Tribe/Aggregator/Admin_Bar.php]: Use di52 rather than a static singleton?

File: src/admin-views/aggregator/tabs/import-form.php [hunk 4]
```diff
@@ -46,7 +46,13 @@
 			$option->upsell = isset( $option->upsell ) ? $option->upsell : false;
 
 			$option->is_selected = false;
-			if ( ! empty( $_GET['ea-auth'] ) && $_GET['ea-auth'] === $option->id ) {
+
+			if (
+				// Used on the EA Authorization
+				tribe_get_request_var( 'ea-auth', false ) === $option->id ||
```
Comment (borkweb) [inline src/admin-views/aggregator/tabs/import-form.php]: Move the `||` to prefix the next line

File: src/Tribe/Aggregator/Admin_Bar.php (hunk 3 above)
Comment (bordoni) [reply src/Tribe/Aggregator/Admin_Bar.php]: For the most part we don't have di52 on Aggregator, so I would rather not do it on this MR.

File: src/Tribe/Aggregator.php (hunk 2 above)
Comment (bordoni) [reply src/Tribe/Aggregator.php]: Same as above.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Admin_Bar.php:35](https://github.com/moderntribe/the-events-calendar/blob/5f5d64519880004a1e3da782ed890e5c8c8104e9/src/Tribe/Aggregator/Admin_Bar.php#L35)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Admin_Bar.php (hunk 1 above)
Comment (ggwicz) [reply src/Tribe/Aggregator/Admin_Bar.php]: @bordoni this is the last thing you need to fix before this can merge — looking good otherwise!

Comment (bordoni) [reply src/Tribe/Aggregator/Admin_Bar.php]: This was fixed.

**Pull request merged**